### PR TITLE
Initial developement of components, new config parser, action on hosts, and host predicates

### PR DESCRIPTION
**Pull request by acmnu**
This is initial PR wich combine the following changes:

* New parser for config.
* UI to component functionality
* New lists view in UI
* Action on Host functionality
* Predicate Solver for host functional of variant element.

### CODE REVIEW

**Review by grey-kristy — APPROVED** [review]

File: assemble/base/requirements-base.txt [hunk 1]
```diff
@@ -13,9 +13,11 @@ jinja2
 markdown
 mitogen
 pyyaml
+ruyaml
```
Comment (dgusakov) [inline assemble/base/requirements-base.txt:16]: Looks like a duplicate with line 20

**Comment by acmnu** [comment]
bender build

**Comment by BenderArenadata** [comment]
Hey, @acmnu! Build has been launched!

**Comment by acmnu** [comment]
bender build

**Comment by BenderArenadata** [comment]
Hey, @acmnu! Build has been launched!

**Pull request merged**